Sources/ParseSwift/Authentication/3rd Party/ParseLDAP.swift

```diff
@@ -200,6 +207,13 @@ public extension ParseLDAP {
     #endif
 }
 
+extension ParseLDAP: ParseAuthenticationDelegate {
+	public func restoreAuthentication(withAuthData authData: [String: String]) -> Bool {
```

Comment:
This should be a requirement in `ParseAuthentication` like https://github.com/parse-community/Parse-Swift/blob/3852a93ed00e532c6214086700fb98e5eb4bd03f/Sources/ParseSwift/Authentication/Protocols/ParseAuthentication.swift#L42-L52

and then have the convenience/default implementation that you have. Similar to: https://github.com/parse-community/Parse-Swift/blob/3852a93ed00e532c6214086700fb98e5eb4bd03f/Sources/ParseSwift/Authentication/Protocols/ParseAuthentication.swift#L164-L169

When an auth type wants to change from the default, they will implement it